packages/preview/modern-cug-thesis/0.2.1/template/thesis.typ

```diff
@@ -0,0 +1,770 @@
+// 在线包管理器模式
+// #import "@preview/modern-cug-thesis:0.2.1": documentclass, indent
+// 本地模式
+#import "../lib.typ": documentclass, indent
```

Comment:
Because this is the version that will be published, it would be nice to use the correct import by default, so that people have a working document when their project is created from the template.

```suggestion
#import "@preview/modern-cug-thesis:0.2.1": documentclass, indent
```